hawkbit-device-simulator/src/test/java/org/eclipse/hawkbit/simulator/AuthenticatedWebSecurityTest.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) 2019 Bosch Software Innovations GmbH and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.hawkbit.simulator;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static org.springframework.security.test.web.servlet.request.SecurityMockMvcRequestPostProcessors.httpBasic;
+
+import io.qameta.allure.Description;
+import org.eclipse.hawkbit.simulator.http.BasicAuthProperties;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.TestPropertySource;
+
+@TestPropertySource(properties = {BasicAuthProperties.CONFIGURATION_ENABLED_PROPERTY + " = true"})
+public class AuthenticatedWebSecurityTest extends DdiWebSecurityTest {
```

Comment:
Please add Allure annotations to the test class instead of the abstract base class

Reply:
Done